public/react-services/saved-objects.js

```diff
@@ -183,8 +200,11 @@ export class SavedObject {
         params
       );
 
-      if (type === 'index-pattern')
-        await this.refreshFieldsOfIndexPattern(id, params.attributes.title, fields);
+      if(satisfyKibanaVersion('<7.11')){
```

Comment:
Could this be:
`if(satisfyKibanaVersion('<7.11') && type === 'index-pattern'){
`

Reply:
Yes, I prefered to nest the `if`s instead a complex condition.